### PR TITLE
Add v0.23.1 and v0.24.0 (as LTS) to releases.md

### PR DESCRIPTION
**Pull request by khrm**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [x] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/kind misc

**Review by dibyom — APPROVED** [review]

File: releases.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ Further documentation available:
 
 ### v0.23
 
-- **Latest Release**: [v0.23.0][v0-23-0] (2023-03-02) ([docs][v0-23-0-docs], [examples][v0-23-0-examples])
+- **Latest Release**: [v0.23.1][v0-23-1] (2023-04-03) ([docs][v0-23-1-docs], [examples][v0-23-1-examples])
```
Comment (savitaashture) [inline releases.md:45]: I believe now latest release is v0.24.0 

Can you add that info as well

Comment (khrm) [reply releases.md:52]: Done.

**Pull request merged**